icu4c/source/test/intltest/usettest.cpp

```diff
@@ -3984,3 +3985,31 @@ void UnicodeSetTest::TestDeepPattern() {
     assertTrue("[a[a[a...1000s...]]] -> error", errorCode.isFailure());
     errorCode.reset();
 }
+
+void UnicodeSetTest::TestEmptyString() {
+    IcuTestErrorCode errorCode(*this, "TestEmptyString");
+    // Starting with ICU 69, the empty string is allowed in UnicodeSet. ICU-13702
+    UnicodeSet set(u"[{}]", errorCode);
+    if (!assertSuccess("set from pattern with {}", errorCode)) { return; }
+    assertTrue("set from pattern with {}", set.contains(u""));
+    assertFalse("remove empty string", set.remove(u"").contains(u""));
+    assertTrue("add empty string", set.add(u"").contains(u""));
```

Comment:
Would also be good to test .isEmpty/.length of the set when the string is or is not in the set.

Reply:
done in a couple of places